src/object/schema_manager.c

```diff
@@ -2168,6 +2169,89 @@ sm_downcase_name (const char *name, char *buf, int maxlen)
   intl_identifier_lower (name, buf);
 }
 
+/* AUTO_INCREMENT_SERIAL_NAME_MAX_LENGTH is not a target. 
+ * Trigger and serial names can be longer than DB_MAX_IDENTIFIER_LENGTH.
+ */
+int
+sm_user_specified_name (const char *name, const char *owner_name, char *buf, int buf_size)
+{
+  char *dot = NULL;
+  const char *name_p = NULL;
+  const char *owner_name_p = NULL;
+  char *current_user_name = NULL;
+  char qualifier_name[DB_MAX_USER_LENGTH] = { '\0' };
+  char user_specified_name[DB_MAX_IDENTIFIER_LENGTH] = { '\0' };
+  int name_size = 0;
+  int error = NO_ERROR;
+
+  if (name == NULL || name[0] == '\0')
+    {
+      er_set (ER_WARNING_SEVERITY, ARG_FILE_LINE, ER_SM_INVALID_NAME, 1, name);
+      return er_errid ();
+    }
+
+  memset (buf, 0, buf_size);
+
+  dot = (char *) strchr (name, '.');
+  if (dot)
+    {
+      name_size = intl_identifier_lower_string_size (name);
+      assert (name_size < buf_size);
+      assert (name_size < DB_MAX_IDENTIFIER_LENGTH);
+
+      intl_identifier_lower (name, buf);
+
```

Comment:
How about replacing it like below?
`sm_downcase_name (name, buf, buf_size);`

Reply:
Okay, I'll change it.